File: test/e2e/test-files/workspaces/performance/multiplyParquet.r
library(arrow)
library(dplyr)

Sys.setenv(TZ='GMT')
df3 <- read_parquet(file.path(getwd(), "data-files", "20x1000", "parquet1kx20.parquet"))
df3_large <- df3[rep(seq_len(nrow(df3)), times = 1000), ]

File: test/e2e/test-files/workspaces/performance/multiplyParquet.py
import pandas as pd
import os

data_file_path = os.path.join(os.getcwd(), 'data-files', '20x1000', 'parquet1kx20.parquet')
df = pd.read_parquet(data_file_path, engine='pyarrow')
df_large = pd.concat([df] * 1000, ignore_index=True)